iac-extensions/arm/src/test/resources/checks/UnencryptedCloudServicesCheck/Compute_disk_and_snapshots.bicep

```
@@ -0,0 +1,146 @@
+// Noncompliant@+1 {{Omitting "encryption.diskEncryptionSetId", "encryptionSettingsCollection" or "securityProfile.secureVMDiskEncryptionSetId" enables clear-text storage. Make sure it is safe here.}}
+resource sensitiveDiskEncryptionSetIdEncryptionSettingsCollectionAndSecureVMDiskEncryptionSetIdAreNotSet 'Microsoft.Compute/disks@2022-07-02' = {
```

```suggestion
resource noncompliant1 'Microsoft.Compute/disks@2022-07-02' = {
```
Suggestion, no need to keep the default decompiled symbolic name, it is not really readable and we already have the full information in the name of the resource.
I usually put `noncompliantX` / `compliantX` instead, so that we quickly identify in which case we are and then can read it if we want to know more.